service/key/key.go

```diff
@@ -55,6 +55,10 @@ func HasClusterVersion(customObject awstpr.CustomObject) bool {
 	}
 }
 
+func IngressDomain(customObject awstpr.CustomObject) string {
+	return customObject.Spec.Cluster.Kubernetes.IngressController.Domain
+}
```

Comment:
I like that you've created a new key function for this.

IMO we should create a key function if we think a value is needed multiple times. If we only need it once I'm OK with getting the value from the custom object.

Reply:
Makes sense, fixed